public/offline.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <link rel="icon" href="favicon.ico"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="theme-color" content="#000000"/>
+    <meta name="description" content="Web site created using create-react-app"/>
+
+    <base href="/"/>
+    
+    <link rel="apple-touch-icon" href="/logo192.png"/>
+    <link rel="manifest" href="/manifest.json"/>
+    <link href="/static/fonts/montserrat.css" type="text/css" rel="stylesheet" />
+
+    <title>Zeeguu</title>
+
+<style>
+  .contentDiv {
+    display: flex; 
+    flex-direction: column;
+    align-items: center;
+    font-family: 'Monsterrat', sans-serif;
+  }
+
+  .capitalizeBold {
+    text-transform: capitalize;
+    font-weight: bold;
+  }
+</style>
+
+<body>
+<div id="offlineRoot" class="contentDiv">
+  <img src="/logo192.png"> 
+  <h1>You seem to be offline</h1>
+  <h2>Please return to Zeeguu when you are back online!</h2>
+  <div id="wordsList">Loading cached words...</div>
+</div>
+
```

Comment:
@mircealungu When we reviewed this in person, I noticed there wasn't a cloasing </head> tag, is there a reason for this?